### DISCUSSION TITLE
How can i do it in free version?

### DISCUSSION THREAD

**Discussion by prashanta0234** [xyflow/xyflow#3484] [Q&A] [answered]
![image (1)](https://github.com/wbkd/react-flow/assets/86629940/288fad17-ede8-46f9-8986-c1eb8c59d5d6)

**Comment by bcakmakoglu** [comment, marked as answer]
To begin with, there is no "non-free" version.
React Flow is free to use, paying for a pro-subscription gives you access to pro *examples*.
There is no library code hidden behind a paywall or anything.

To get to your "question" (which you haven't really asked besides a screenshot) - [maybe this example might help.](https://reactflow.dev/docs/examples/edges/floating-edges/)
